### DISCUSSION TITLE
Code

### DISCUSSION THREAD

**Discussion by Nick180121** [alexanderlavrushko/BLE-HUD-navigation-ESP32#24] [General]
Hello, first of all I am sorry for the lost question, which I will ask you here, but I am absolutely new to this. My Problem is that I can not start the program. Do I need both Programms, so the one for the arduino and also for the nRF? And another question is if the nRF is on the ESP32 or do I have to order this one too? 

Dear regard and thank you in advance for your help

**Comment by alexanderlavrushko** [comment]
Hello @Nick180121, primary project is ESP32. It works with 2 configurations - see [Supported ESP32 modules](https://github.com/alexanderlavrushko/BLE-HUD-navigation-ESP32#supported-esp32-modules) in Readme. The easiest way to start is ESP32 TTGO T-Display, but you will need to change a few lines of code to enable it, it's described in Readme: [TTGO T-Display](https://github.com/alexanderlavrushko/BLE-HUD-navigation-ESP32#ttgo-t-display).

P.S.: nRF part and ESP32 part are independent from each other. nRF is just my little experiment with reprogramming of cheap watch, and it works only with that specific model (written in the beginning of .ino file), which is impossible to find now. So I don't recommend trying it.
